src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionsLocalAnnotations.java

```diff
@@ -0,0 +1,12 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+public class InputDesignForExtensionsLocalAnnotations
+{
+    public @interface Rule {
```

Comment:
It is better to keep inputs independent on any other file. We do not need good design in this files, the aglier the better.